v0/destinations/lytics/data/LYTICSPageScreenConfig.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "destKey": "_e",
```

Comment:
Please review [this comment about the usage of the special key `"_e"` for custom events](https://github.com/rudderlabs/rudder-sdk-js/pull/144#discussion_r527897323)

Reply:
Done